src/renderer/src/renderer/debug/axes.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <array>
+#include <memory>
+
+#include "renderer/scene_object.h"
+#include "renderer/vertexarray.h"
+
+namespace debug {
+
+class Axes : public SceneObject {
+ public:
+	explicit Axes();
+	[[nodiscard]] const VertexArray& vertex_array() const { return *vao_; }
```

Comment:
why getter name and variable not consistant?

Reply:
why not?